pkg/util/iptables/iptables.go

```diff
@@ -328,10 +328,11 @@ func (runner *runner) checkRuleWithoutCheck(table Table, chain Chain, args ...st
 
 	// Sadly, iptables has inconsistent quoting rules for comments.
 	// Just unquote any arg that is wrapped in quotes.
-	argsCopy := make([]string, len(args))
-	copy(argsCopy, args)
-	for i := range argsCopy {
-		unquote(&argsCopy[i])
+	var argsCopy []string
+	for i := range args {
+		tmpField := args[i]
+		unquote(&tmpField)
+		argsCopy = append(argsCopy, strings.Fields(tmpField)...)
```

Comment:
I don't understand why this is correct.  You're unpacking single args like "foo bar bat" into multiple args.  How is this correct?  I'm not saying it's not, but I have no way to repro it, so I can't understand it.  can you show me an example output that this parses properly that was wrong before?